src/include/btree_inline.h

```diff
@@ -1512,8 +1512,9 @@ __wt_page_del_active(WT_SESSION_IMPL *session, WT_REF *ref, bool visible_all)
     WT_ORDERED_READ(prepare_state, page_del->prepare_state);
     if (prepare_state == WT_PREPARE_INPROGRESS || prepare_state == WT_PREPARE_LOCKED)
         return (true);
-    return (visible_all ? !__wt_txn_visible_all(session, page_del->txnid, page_del->timestamp) :
-                          !__wt_txn_visible(session, page_del->txnid, page_del->timestamp));
+    return (visible_all ?
+        !__wt_txn_visible_all(session, page_del->txnid, page_del->durable_timestamp) :
+        !__wt_txn_visible(session, page_del->txnid, page_del->timestamp));
```

Comment:
Should this be the durable timestamp in both cases or only for visible_all? If it was intentional I'm happy to go along, just want to make sure it's not an oversight.

Reply:
The visibility of a transaction is always compared against the `commit timestamp`, whereas the stability or obsoleteness of an update is decided against the `durable timestamp` of the operation. Due to this reason, I changed it to `durable timestamp` only for the `visible_all` case.